spec/fabricators/user_fabricator.rb

```diff
@@ -4,4 +4,5 @@
   email { FFaker::Internet.unique.email }
   password { 'password123' }
   password_confirmation { 'password123' }
+  confirmed_at { FFaker::Time.datetime }
```

Comment:
```suggestion
  confirmed_at { 1.month.ago }
```

I recommend using a date in the past as we do not expect a password to "will be confirmed in the future" 🙈